serverlessworkflow/modules/ROOT/pages/job-services/quarkus-extensions.adoc

```diff
@@ -0,0 +1,245 @@
+
+. Add the `kogito-addons-quarkus-jobs-knative-eventing` extension to your Quarkus Workflow Project using any of the following alternatives.
+
+[tabs]
+====
+Manually::
++
+[source,xml]
+----
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-addons-quarkus-jobs-knative-eventing</artifactId>
+</dependency>
+----
+Apache Maven::
++
+[source,shell]
+----
+mvn quarkus:add-extension -Dextensions="kogito-addons-quarkus-jobs-knative-eventing"
+----
+Quarkus CLI::
++
+[source,shell]
+----
+quarkus extension add kogito-addons-quarkus-jobs-knative-eventing
+----
+====
+
+[start=3]
+. Add the following configurations to the `application.properties` file of your project.
+
+[source,properties]
+----
+mp.messaging.outgoing.kogito-job-service-job-request-events.connector=quarkus-http
+mp.messaging.outgoing.kogito-job-service-job-request-events.url=${K_SINK:http://localhost:8280/v2/jobs/events}
+mp.messaging.outgoing.kogito-job-service-job-request-events.method=POST
+----
+
+[NOTE]
+====
+The `K_SINK` environment variable is automatically generated by the combination of the knative ecosystem and the SinkBinding definition that will be automatically generated in the `kogito.yml` file.
+
+If this variable is not present, the default value `http://localhost:8280/v2/jobs/events` is used instead, this can be useful in development environments if you are executing the Job Service as a standalone service.
+====
+
+[start=2]
+. Build your project and locate the automatically generated `kogito.yml` and `knative.yml` files in the `/target/kubernetes` directory of your project, xref:eventing/consume-produce-events-with-knative-eventing.adoc#proc-generating-kn-objects-build-time[see].
+
+[source,shell]
+----
+mvn clean install
+----
+
+[start=3]
+. Use the generated files to deploy your workflow application in the Kubernetes cluster using the following commands:
+
+[source, bash]
+----
+kubectl apply -f /target/kogito.yml
+
+kubectl apply -f /target/knative.yml
+----
+
+You can see a full example of this interaction mode configuration in the xref:use-cases/timeout-showcase-example.adoc#execute-quarkus-project-standalone-services[Quarkus Workflow Project with standalone services] example project.
+
+[#kogito-addons-quarkus-jobs-messaging]
+== Kafka messaging interaction
+
+To interact with the Job Service by sending cloud events over the kafka messaging system you must follow these steps:
+
+. Be sure that you have read the xref:eventing/consume-producing-events-with-kafka.adoc[Consuming and producing events with Kafka] guide, and you have configured the project accordingly.
+
+. Add the `quarkus-smallrye-reactive-messaging-kafka` and `kogito-addons-quarkus-jobs-messaging` extensions to your Quarkus Workflow Project using any of the following alternatives.
```

Comment:
```suggestion
. Add the `quarkus-smallrye-reactive-messaging-kafka` and `kogito-addons-quarkus-jobs-messaging` extensions to your Quarkus Workflow project using any of the following alternatives.
```

Reply:
@kaldesai  I think "Quarkus Workflow Project" is being used all the time, and I think is good, represents a particular subject in this context ¿? Not a problem to change, but why is "Project" not good?